manifests/compute_node/config.pp

```diff
@@ -20,18 +20,20 @@
 # http://www.apache.org/licenses/LICENSE-2.0.html
 #
 class one::compute_node::config (
-  $networkconfig     = $one::kickstart_network,
-  $partitions        = $one::kickstart_partition,
-  $rootpw            = $one::kickstart_rootpw,
-  $data              = $one::kickstart_data,
-  $kickstart_tmpl    = $one::kickstart_tmpl,
-  $preseed_data      = $one::preseed_data,
-  $ohd_deb_repo      = $one::preseed_ohd_deb_repo,
-  $debian_mirror_url = $one::preseed_debian_mirror_url,
-  $preseed_tmpl      = $one::preseed_tmpl,
-  $libvirtd_cfg      = $one::libvirtd_cfg,
-  $libvirtd_source   = $one::libvirtd_source,
-  $libvirtd_srv      = $one::libvirtd_srv
+  $networkconfig           = $one::kickstart_network,
+  $partitions              = $one::kickstart_partition,
+  $rootpw                  = $one::kickstart_rootpw,
+  $data                    = $one::kickstart_data,
+  $kickstart_tmpl          = $one::kickstart_tmpl,
+  $preseed_data            = $one::preseed_data,
+  $ohd_deb_repo            = $one::preseed_ohd_deb_repo,
+  $debian_mirror_url       = $one::preseed_debian_mirror_url,
+  $preseed_tmpl            = $one::preseed_tmpl,
+  $libvirtd_cfg            = $one::libvirtd_cfg,
+  $libvirtd_source         = $one::libvirtd_source,
+  $libvirtd_srv            = $one::libvirtd_srv,
+  $oneadmin_sudoers_file   = $one::oneadmin_sudoers_file,
+  $sudoers_imaginator_file = $one::sudoers_imaginator_file
```

Comment:
currently we depend on puppetlabs-stdlib > 2.2.0 and validate_absolute_path appeared in version 2.3.0. So we could either trick something using regexp (though the path regexp is a bit nasty: https://github.com/puppetlabs/puppetlabs-stdlib/blob/2.3.x/lib/puppet/parser/functions/validate_absolute_path.rb#L44) or bump the required version of stdlib which breaks compat with puppet 2.6.